Lundii/HackerBay-Interview-Round-1
# #168603471 Setup eslint and airnub

#### what does this PR do?
- It sets up javascript linting using eslint and airnub style guide

#### how can it be tested
- fetch or pull this branch
- run `npm install` to get all required package.
- if you use vscode editor, install the eslint extension.
- type 
 ```
   var me = "string"
 ```
to see the linting in action.
- you can also run `npm run lint` to review the code and fix any auto-fixable style guide rule violation

#### pivotal story link
[#168603471](https://www.pivotaltracker.com/story/show/168603471)
